Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'activerecord-import'
 gem 'coffee-script-source', '1.8.0'
 
 gem 'devise'
+gem 'devise_invitable'
+gem 'cancan'
```

Comment:
We may want to consider using `cancancan`, since the `cancan` project and gem are no longer in development. `cancancan` should have same interface.

https://github.com/CanCanCommunity/cancancan